BioPhoton/rx-angular
# State documentation for web page

## State documentation for web page

### Merge After

#235 - changes in GitHub `.md` files needed to correctly display content on web page.

### What Problem/Issue is it trying to solve?

#217 - Docs checkbox :)

### Change List

- [Docsify](https://docsify.js.org/#/) generator added that will parse `.md` files to `html`.
- Docs skeleton pages added to `docs/web` folder. `.md` files form the `libs/state/docs` are imported to those pages.
- Michael added [netlifyCMS](https://www.netlifycms.org/) that will help us manage `.md` files better. Setup files are in `docs/admin`.
- Netlify collections were changed. Now we have separate collections for each subfolder in `docs/web` and `libs/{libName}/docs`


### How to Test it

- Checkout branch
- run `yarn run nx serve-docsify docs` and open localhost:3000 to see how web page for the docs will look like
- run `yarn run nx serve-admin docs` and open localhost:5000 to see how admin page for `.md` files look like
